src/components/styled-list/components/StyledListItem.vue

```diff
@@ -0,0 +1,129 @@
+<template>
+    <li
+        class="vs-styled-list__item"
+        data-test="vs-styled-list__item"
+    >
+        <VsRow>
+            <VsCol
+                v-if="props.variant === 'image'"
+                cols="12"
+                md="2"
+            >
+                <VsImg :src="props.imageSrc" />
+            </VsCol>
+            <VsCol :md="props.variant === 'image' ? '10' : '12'">
+                <VsImg
+                    v-if="props.variant === 'image-horizontal'"
+                    :src="props.imageSrc"
+                />
+
+                <VsHeading level="3">
+                    {{ heading }}
+                </VsHeading>
+
+                <VsRichTextWrapper>
+                    <slot />
```

Comment:
Slots need comments too - take a look at https://visitscotland.atlassian.net/wiki/spaces/DE/pages/159088753/Vue

Reply:
I've added comments to all slots